swift-sdk/Internal/LocalStorage.swift

```diff
@@ -94,19 +94,36 @@ struct LocalStorage: LocalStorageProtocol {
     
```

Comment:
You need to read and write from keychain here.

Reply:
Maybe adding some tests here to catch problems like this.

Reply:
Added a general test to verify presence in the keychain, and the lack of presence in the user defaults.